hack/generator/pkg/astmodel/generation_consts.go

```diff
@@ -8,7 +8,14 @@ package astmodel
 var (
 	CodeGenerationComments []string = []string{
 		// Note that this format is actually an official specification in go:  https://github.com/golang/go/issues/13560
-		"// Code generated by k8s-infra. DO NOT EDIT.", // TODO: Update this when the generated is moved/renamed
+		"// Code generated by azure-service-operator. DO NOT EDIT.", // TODO: Update this when the generated is moved/renamed
```

Comment:
Maybe remove the TODO? Although I guess we still want to update it if it's moved again.

Reply:
I agree - remove TODO. I think the TODO was in reference to what @theunrepentantgeek just did (i.e. that it shouldn't say k8sinfra)

Reply:
Nah, that TODO has been there for ages. I'll remove it though.